dice.py

```diff
@@ -24,4 +25,4 @@ def rollingDice():
             print("You won!!!")
             break
         else:
-            sum = rollingDice()
```

Comment:
The code works, is easy to understand and does everything it needs to do:)
I'd suggest you to leave comments within your code in the future when writing more complex programs.